pkgs/tools/misc/yubikey-neo-manager/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, makeWrapper, python27Packages, pyside, pycrypto
+, libykneomgr, yubikey-personalization, libu2f-host }:
+
+python27Packages.buildPythonPackage rec {
+  namePrefix = "";
+  name = "yubikey-neo-manager-${version}";
+  version = "1.4.0";
+  src = fetchurl {
+    url = "https://developers.yubico.com/yubikey-neo-manager/Releases/${name}.tar.gz";
+    sha256 = "1isxvx27hk0avxwgwcwys2z8ickfs816ii1aklvmi09ak1rgrf1g";
+  };
+
+  propagatedBuildInputs = [ pyside pycrypto ];
+  patches = [ ./fix-pyside-requirement.diff ];
+
+  # aid ctypes load_libary()
+  makeWrapperArgs = [
+    "--set LD_PRELOAD '${libykneomgr}/lib/libykneomgr.so ${yubikey-personalization}/lib/libykpers-1.so ${libu2f-host}/lib/libu2f-host.so'"
+  ];
```

Comment:
One of the things that blew my mind when reading Eelcos original paper was that Nix would automatically scan all files from the output for store paths and add them as runtime dependencies. Once you have that piece of knowledge, adding store paths to a wrapper is explicit enough IMO :)

Anyway, thanks for the review.